File: CS/TwoSchedulers/Form1.cs
using System;
using System.Windows.Forms;
using DevExpress.XtraScheduler;

namespace TwoSchedulers
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
            for (int i = 0; i < 10; i++)
                schedulerStorage1.Resources.Add(new Resource(i, "Res" + i.ToString()));
        }

        private void schedulerControl1_VisibleIntervalChanged(object sender, EventArgs e)
        {
            SetStart(schedulerControl2, schedulerControl1);            
        }

        private void schedulerControl2_VisibleIntervalChanged(object sender, EventArgs e)
        {
            SetStart(schedulerControl1, schedulerControl2);            
        }

        private static void SetStart(SchedulerControl firstScheduler, SchedulerControl secondScheduler)
        {
            firstScheduler.Start = secondScheduler.Start;
        }
        
        private void schedulerControl1_CustomDrawResourceHeader(object sender, CustomDrawObjectEventArgs e)
        {
            SetFirstVisibleResourceIndex(schedulerControl2, schedulerControl1);
            SetResourcesPerPage(schedulerControl2, schedulerControl1);
        }

        private void schedulerControl2_CustomDrawResourceHeader(object sender, CustomDrawObjectEventArgs e)
        {
            SetFirstVisibleResourceIndex(schedulerControl1, schedulerControl2);
            SetResourcesPerPage(schedulerControl1, schedulerControl2);
        }

        private static void SetFirstVisibleResourceIndex(SchedulerControl firstScheduler, SchedulerControl secondScheduler)
        {
            if (firstScheduler.ActiveView.FirstVisibleResourceIndex != secondScheduler.ActiveView.FirstVisibleResourceIndex)
                firstScheduler.ActiveView.FirstVisibleResourceIndex = secondScheduler.ActiveView.FirstVisibleResourceIndex;
        }

        private static void SetResourcesPerPage(SchedulerControl firstScheduler, SchedulerControl secondScheduler)
        { 
            if(firstScheduler.ActiveView.ResourcesPerPage != secondScheduler.ActiveView.ResourcesPerPage)
                firstScheduler.ActiveView.ResourcesPerPage = secondScheduler.ActiveView.ResourcesPerPage;
        }
    }
}

File: VB/TwoSchedulers/Form1.vb
Imports System
Imports System.Windows.Forms
Imports DevExpress.XtraScheduler

Namespace TwoSchedulers
    Partial Public Class Form1
        Inherits Form

        Public Sub New()
            InitializeComponent()
            For i As Integer = 0 To 9
                Dim resource As Resource = schedulerStorage1.CreateResource(i)
                resource.Caption = "Resource" & i
                schedulerStorage1.Resources.Add(resource)
            Next i
        End Sub
        Private Sub schedulerControl1_VisibleIntervalChanged(ByVal sender As Object, ByVal e As EventArgs) Handles schedulerControl1.VisibleIntervalChanged
            SetStart(schedulerControl2, schedulerControl1)
        End Sub

        Private Sub schedulerControl2_VisibleIntervalChanged(ByVal sender As Object, ByVal e As EventArgs) Handles schedulerControl2.VisibleIntervalChanged
            SetStart(schedulerControl1, schedulerControl2)
        End Sub

        Private Shared Sub SetStart(ByVal firstScheduler As SchedulerControl, ByVal secondScheduler As SchedulerControl)
            firstScheduler.Start = secondScheduler.Start
        End Sub

        Private Sub schedulerControl1_CustomDrawResourceHeader(ByVal sender As Object, ByVal e As CustomDrawObjectEventArgs) Handles schedulerControl1.CustomDrawResourceHeader
            SetFirstVisibleResourceIndex(schedulerControl2, schedulerControl1)
            SetResourcesPerPage(schedulerControl2, schedulerControl1)
        End Sub

        Private Sub schedulerControl2_CustomDrawResourceHeader(ByVal sender As Object, ByVal e As CustomDrawObjectEventArgs) Handles schedulerControl2.CustomDrawResourceHeader
            SetFirstVisibleResourceIndex(schedulerControl1, schedulerControl2)
            SetResourcesPerPage(schedulerControl1, schedulerControl2)
        End Sub

        Private Shared Sub SetFirstVisibleResourceIndex(ByVal firstScheduler As SchedulerControl, ByVal secondScheduler As SchedulerControl)
            If firstScheduler.ActiveView.FirstVisibleResourceIndex <> secondScheduler.ActiveView.FirstVisibleResourceIndex Then
                firstScheduler.ActiveView.FirstVisibleResourceIndex = secondScheduler.ActiveView.FirstVisibleResourceIndex
            End If
        End Sub

        Private Shared Sub SetResourcesPerPage(ByVal firstScheduler As SchedulerControl, ByVal secondScheduler As SchedulerControl)
            If firstScheduler.ActiveView.ResourcesPerPage <> secondScheduler.ActiveView.ResourcesPerPage Then
                firstScheduler.ActiveView.ResourcesPerPage = secondScheduler.ActiveView.ResourcesPerPage
            End If
        End Sub
    End Class
End Namespace
